client/container.go

```diff
@@ -429,6 +429,9 @@ func (c *Client) ContainerDelete(ctx context.Context, prm PrmContainerDelete) er
 		signer = c.defaultSigner()
 	}
 
+	if signer.Scheme() != neofscrypto.ECDSA_DETERMINISTIC_SHA256 {
+		return fmt.Errorf("%w: expected ECDSA_DETERMINISTIC_SHA256 scheme", neofscrypto.ErrIncorrectSigner)
```

Comment:
I consider we may derive the error and put it inside `neofscrypto` package, like:
```go
var ErrEcdsaDeterministicSha256 = fmt.Errorf("%w: expected ECDSA_DETERMINISTIC_SHA256 scheme", neofscrypto.ErrIncorrectSigner)
```
and reuse in this and another places.

PS. check linters

Reply:
I've specifically wanted to make `ErrIncorrectSigner` generic enough for various cases, but I've dropped some duplicates from the `client` package.